examples/rust/examples/maker/maker.rs

```diff
@@ -143,12 +163,14 @@ async fn run<P: JsonRpcClient + 'static>(
     let signer =
         SignerMiddleware::new_with_provider_chain(Arc::clone(&provider), settings.wallet.clone())
             .await
-            .unwrap();
+            .ok()?;
 
     // Seaport 1.5 contract address
+    // Note: We allow the unchecked unwrap here, since this address will always parse correctly.
     let seaport_contract_address = "0x00000000000000ADc04C56Bf30aC9d3c0aAF14dC"
         .parse::<Address>()
         .unwrap();
```

Comment:
```suggestion
        .ok?;
```

Reply:
This is never expected to fail so we should never silently ignore this, we want this to panic since its a hardcoded address (i.e. we can only fix it up updating code).